Source/WebCore/style/CustomPropertyRegistry.h

```diff
@@ -50,14 +50,20 @@ class CustomPropertyRegistry {
 
     const RenderStyle& initialValuePrototypeStyle() const;
 
+    void invalidatePropertiesWithViewportUnits(Document&);
+
+    enum class ViewportUnitDependency : bool { No, Yes };
+    enum class ParseInitialValueError : uint8_t { NotComputationallyIndependent, DidNotParse };
```

Comment:
It's short enough to be a `bool`.

Reply:
My brain thought `bool` but my fingers typed `uint8_t`. :-)

Reply:
We have some in the codebase, but indeed we are not consistent about 2 values "non boolean" enum, neither the coding style. I don't know what we should do about it 😅